.github/workflows/check-excluded-packages.yml

```diff
@@ -45,13 +48,17 @@ jobs:
           cargo-targets: "${{ steps.format_output.outputs.targets }}"
 
       - name: Check excluded packages
+        env:
+          RUSTC_WRAPPER: ""
+          RUSTC_WORKSPACE_WRAPPER: sccache
         run: |
           packages="${{ steps.format_output.outputs.packages }}"
           for p in ${packages[@]}
           do
-            echo "Compiling package $p..."
+            echo "Checking package $p..."
             pushd "$p"
-            cargo check --all-targets --all-features
+            cargo fmt --all
```

Comment:
great catch! in this form it would correct code - I have changed that to 'check' mode which in case of any format-problems returns code `1` which should stop pipeline